cli/dcoscli/node/main.py

```diff
@@ -482,15 +484,71 @@ def _log(follow, lines, leader, slave):
 
     if lines is None:
         lines = 10
+
     lines = util.parse_int(lines)
 
-    mesos_files = _mesos_files(leader, slave)
+    try:
+        _dcos_log(follow, lines, leader, slave)
+        return 0
+    except DCOSException as e:
```

Comment:
ah i see. if the new API exists though, there shouldn't be an exception unless the api has a bug or a problem connecting with dcos_url, right?